packages/paste-website/src/pages/components/chat-composer/index.mdx

```diff
@@ -62,122 +81,426 @@ export const getStaticProps = async () => {
+              <Avatar name="Lauren Gardner" size="sizeIcon20" />
+              Lauren Gardner ・ 4:30 PM
+            </ChatMessageMetaItem>
+          </ChatMessageMeta>
+        </ChatMessage>
+      ),
+    },
+  );
+  const [message, setMessage] = React.useState("");
+  const [mounted, setMounted] = React.useState(false);
+  const loggerRef = React.useRef<HTMLDivElement>(null);
+  const scrollerRef = React.useRef<HTMLDivElement>(null);
+  React.useEffect(() => {
+    setMounted(true);
+  }, []);
+  React.useEffect(() => {
+    if (!mounted || !loggerRef.current) return;
+    scrollerRef.current?.scrollTo({ top: loggerRef.current.scrollHeight, behavior: "smooth" });
+  }, [chats, mounted]);
+  const handleComposerChange = (editorState): void => {
+    editorState.read(() => {
+      const text = $getRoot().getTextContent();
+      setMessage(text);
+    });
+  };
+  const submitMessage = (): void => {
+    if (message === "") return;
+    push(createNewMessage(message));
+  };
+  const editorInstanceRef = React.useRef<LexicalEditor>(null);
+  return (
+    <Box>
+      <Box ref={scrollerRef} overflowX="hidden" overflowY="auto" maxHeight="size50" tabIndex={0}>
+        <ChatLogger ref={loggerRef} chats={chats} />
+      </Box>
+      <Box
+        borderStyle="solid"
+        borderWidth="borderWidth0"
+        borderTopWidth="borderWidth10"
+        borderColor="colorBorderWeak"
+        columnGap="space30"
+        paddingX="space70"
+        paddingTop="space50"
+      >
+        <ChatComposerContainer>
+          <ChatComposer
+            maxHeight="size10"
+            config={{
+              namespace: "foo",
+              onError: (error) => {
+                throw error;
+              },
+            }}
+            ariaLabel="Message"
+            placeholder="Type here..."
+            onChange={handleComposerChange}
+            editorInstanceRef={editorInstanceRef}
+          >
+            <ClearEditorPlugin />
+            <EnterKeySubmitPlugin onKeyDown={submitMessage} />
+          </ChatComposer>
+          <ChatComposerActionGroup>
+            <Button variant="secondary_icon" size="reset">
+              <AttachIcon decorative={false} title="attach files to the message" />
+            </Button>
+            <Button
+              variant="primary_icon"
+              size="reset"
+              onClick={() => {
+                submitMessage();
+                editorInstanceRef.current?.dispatchCommand(CLEAR_EDITOR_COMMAND, undefined);
+              }}
+            >
+              <SendIcon decorative={false} title="Send" />
+            </Button>
+          </ChatComposerActionGroup>
+        </ChatComposerContainer>
+      </Box>
+    </Box>
+  );
+}:`}
+/>
+
+### Chat Composer with AI Chat Log
+
+Use Chat Composer alongside other Paste components such as [AI Chat Log](/components/ai-chat-log) to build more complex chat UI. For the AI experience be sure to use the `contained` variant.
+
+<StoryPreview
```

Comment:
issue: 2 things here:

1. What loads as the example is different than what the code shows
2. Can you make the Chat Composer only send messages as the user? Right now, it sometimes sends as the bot.